src/crccombine.c

```diff
@@ -0,0 +1,456 @@
+#include <stdint.h>
```

Comment:
Didn't really look through this code, sort of just assuming it works as described in the .h file.